example_test.go

```diff
@@ -18,10 +18,10 @@ import (
 	pb "go.etcd.io/raft/v3/raftpb"
 )
 
-func applyToStore(ents []pb.Entry)    {}
-func sendMessages(msgs []pb.Message)  {}
-func saveStateToDisk(st pb.HardState) {}
-func saveToDisk(ents []pb.Entry)      {}
+func applyToStore(ents []pb.Entry)    {} //revive:disable-line:unused-parameter
+func sendMessages(msgs []pb.Message)  {} //revive:disable-line:unused-parameter
+func saveStateToDisk(st pb.HardState) {} //revive:disable-line:unused-parameter
+func saveToDisk(ents []pb.Entry)      {} //revive:disable-line:unused-parameter
```

Comment:
```suggestion
func applyToStore(_ []pb.Entry)    {}
func sendMessages(_ []pb.Message)  {}
func saveStateToDisk(_ pb.HardState) {}
func saveToDisk(_ []pb.Entry)      {}
```

Reply:
Done - disabled initially as was wondering if we were trying to give some param naming hint in example test.